source/installguide/management-server/_database.rst

```diff
@@ -197,6 +197,17 @@ MySQL. See :ref:`install-database-on-separate-node`.
       cluster management server node IP. If not specified, the local IP
       address will be used.
 
+   -  (Optional) The cloudstack-setup-databases script is responsible for
+      creating the cloudstack databases (cloud, cloud_usage), creating a
+      user (cloud), granting permissions to the user and preparing the tables
+      for the first startup of the management server. There is an option to
+      bypass the two initial steps of creating the databases and granting
+      permissions to the user. This is useful if you don't want to expose your
+      root credentials but still want the database to be prepeared for first
+      start up. These skipped steps will have had to be done manually prior to
+      executing this script. This behaviour can be envoked by passing the
+      --schema-only flag. This flag conflicts with the --deploy-as flag so
+      the two cannot be used together.
 
```

Comment:
Update in the below section also.
http://docs.cloudstack.apache.org/en/latest/installguide/management-server/index.html#install-the-database-on-a-separate-node

Reply:
I don't think it is applicable to this section @sureshanaparti, If you use the --schema-only flag here, you will break the database.

Reply:
> I don't think it is applicable to this section @sureshanaparti, If you use the --schema-only flag here, you won't get the desired result, in this case, adding another management server.

@Spaceman1984 It's not another management server, it is actually DB setup in separate node and there can be cases where user / databases are created already in that node and only schema creation is required.